### PR TITLE
Examples: Add example to submit a `PwCalculation`

### PR DESCRIPTION
**Pull request by sphuber**
The script should run for any setup that has `aiida-quantumespresso` installed, has a daemon running and has at least one `Code` configured with the `default_calc_job_plugin` set to `quantumespresso.pw`. With the `--workchain` flag a `PwBaseWorkChain` is launched instead of the calculation job.

### CODE REVIEW

File: examples/submit_quantumespresso_pw.py [hunk 1]
```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Example script to submit a ``PwCalculation`` or ``PwBaseWorkChain`` over the web API.
+
+Simply make this script executable and execute it. Pass the ``--help`` option for more detailed information.
```
Comment (ltalirz) [inline examples/submit_quantumespresso_pw.py:5]: git stores the executable bit, i.e. if you set it, the user will also have it

**Review by ltalirz — COMMENTED** [review]
This is great, thanks a lot @sphuber !

To make it even more easy to use, instead of describing the setup just in the commit message, could you please add a `README.md` to the folder that includes the steps the user needs to take on the server (ideally even just the sequence of commands) as well as what needs to be done on the client side (e.g. `pip install requests`?

**Comment by sphuber** [comment]
> To make it even more easy to use, instead of describing the setup just in the commit message, could you please add a `README.md` to the folder that includes the steps the user needs to take on the server (ideally even just the sequence of commands) as well as what needs to be done on the client side (e.g. `pip install requests`?

Good point, happy to add complete usage instructions for server/client. Is there an argument against putting it in the docstring of the script itself though instead of a separate README file? It's just that it makes it more self-contained. If somebody were to just copy/send the script, the full description is there.

**Comment by ltalirz** [comment]
> Good point, happy to add complete usage instructions for server/client. Is there an argument against putting it in the docstring of the script itself though instead of a separate README file? It's just that it makes it more self-contained. If somebody were to just copy/send the script, the full description is there.

The argument for the README would be that it will be rendered automatically on github when going to the example folder.
But also on the file system, users will most certainly see it, while I would guess that instructions buried in a python script will probably be found by a smaller percentage of users (some will just try to run the script and see what happens).

**Comment by sphuber** [comment]
@ltalirz created a README with detailed instructions.

**Review by ltalirz — APPROVED** [review]
great, thanks a lot!

**Pull request merged**